### DISCUSSION TITLE
Failed to load resource: net::ERR_CONNECTION_REFUSED

### DISCUSSION THREAD

**Discussion by Hytroxx** [saleor/saleor#10105] [Deployment]
I have deployed saleor on azure virtual machine via docker. Both API and React-Storefront works fine.

Dashboard is not working correctly. It loads but i can not log in - There is error that connection to graphql API is refused, but storefront is making request to API without problems.

![browser](https://user-images.githubusercontent.com/70727125/177347700-0c25f274-6765-418c-bf2e-60b61c601784.PNG)
![docker](https://user-images.githubusercontent.com/70727125/177347752-b22f72ae-81c9-433b-acf0-2d6d979134f9.PNG)

**Comment by Sl8my** [comment]
Hi, did you solve it?

**Comment by dasaintgray** [comment]
i have the same problem
